dipy/workflows/tests/test_align.py

```diff
@@ -20,12 +25,48 @@ def test_reslice():
         out_path = reslice_flow.last_generated_outputs['out_resliced']
         out_img = nib.load(out_path)
         resliced = out_img.get_data()
-        
+
         npt.assert_equal(resliced.shape[0] > volume.shape[0], True)
         npt.assert_equal(resliced.shape[1] > volume.shape[1], True)
         npt.assert_equal(resliced.shape[2] > volume.shape[2], True)
         npt.assert_equal(resliced.shape[-1], volume.shape[-1])
-        
+
+
+def test_slr_flow():
+    with TemporaryDirectory() as out_dir:
+        data_path = get_data('fornix')
+
+        streams, hdr = nib.trackvis.read(data_path)
+        fornix = [s[0] for s in streams]
+
+        f = Streamlines(fornix)
+        f1 = f.copy()
+
+        f1_path = pjoin(out_dir, "f1.trk")
+        save_trk(f1_path, Streamlines(f1), affine=np.eye(4))
+
+        f2 = f1.copy()
+        f2._data += np.array([50, 0, 0])
+
+        f2_path = pjoin(out_dir, "f2.trk")
+        save_trk(f2_path, Streamlines(f2), affine=np.eye(4))
+
+        slr_flow = SlrWithQbxFlow(force=True)
+        slr_flow.run(f1_path, f2_path)
+
+        out_path = slr_flow.last_generated_outputs['out_moved']
+        moved_f2, _ = load_trk(out_path)
+
+        BMD = BundleMinDistanceMetric()
+        nb_pts = 20
+        static = set_number_of_points(f1, nb_pts)
+        moving = set_number_of_points(moved_f2, nb_pts)
+
+        BMD.setup(static, moving)
+        x0 = np.array([0, 0, 0, 0, 0, 0, 1., 1., 1, 0, 0, 0])  # affine
+        bmd_value = BMD.distance(x0.tolist())
```

Comment:
I do not see any `assert ` on this test function. It would be good if you checked whether the output files exist or not.